server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/IdentitySetContract.scala

```diff
@@ -1191,4 +1191,134 @@ trait IdentitySetContract {
       .extract()
       .jsonPath()
       .get(s"methodResponses[0][1].created.$clientId.id")
+
+  @Test
+  def updateShouldAcceptServerSetId(): Unit = {
+    val identityId: String = `given`
+      .header(ACCEPT.toString, ACCEPT_RFC8621_VERSION_HEADER)
+      .body(
+        s"""{
+           |	"using": ["urn:ietf:params:jmap:core", "urn:ietf:params:jmap:submission"],
+           |	"methodCalls": [
+           |		["Identity/get",
+           |			{
+           |				"accountId": "29883977c13473ae7cb7678ef767cbfbaffc8a44a6e463d971d23a65c1dc4af6",
+           |				"ids": null
+           |			}, "c2"
+           |		]
+           |
+           |	]
+           |}""".stripMargin)
+    .when
+      .post
+    .`then`
+      .statusCode(SC_OK)
+      .extract()
+      .jsonPath()
+      .get("methodResponses[0][1].list[0].id")
```

Comment:
Please use a JMAP back reference to only do a single JMAP call :-)

Reply:
I don't think you can use a back ref on an update? Not sure to get the point here

Reply:
Yes maybe not doable...

Reply:
```scala
val identityId = UUID.nameUUIDFromBytes("bob@domain.tld".getBytes(StandardCharsets.UTF_8))
```
to easily get the default server set id

Reply:
Very true!